modules/tinymce/src/core/main/ts/file/Uploader.ts

```diff
@@ -31,13 +31,20 @@ import Tools from '../api/util/Tools';
  * });
  */
 
-export type UploadHandler = (blobInfo: BlobInfo, success: (url: string) => void, failure: (err: string) => void, progress?: (percent: number) => void) => void;
+export interface UploadFailureOptions {
+  remove?: boolean;
+}
+
+export type UploadHandler = (blobInfo: BlobInfo, success: (url: string) => void, failure: (err: string, options?: UploadFailureOptions) => void, progress?: (percent: number) => void) => void;
 
 export interface UploadResult {
   url: string;
   blobInfo: BlobInfo;
   status: boolean;
-  error?: string;
+  error?: {
+    options: UploadFailureOptions;
+    error: string;
```

Comment:
Maybe this should be message since result.error.error looks a bit funny, message would also be a bit more similar to the js Error interface.

Reply:
That makes sense.